src/app/(public-pages)/deals/[id]/page.tsx

```diff
@@ -50,6 +53,17 @@ export default async function DealPage({ params }: { params: { id: string } }) {
   return (
     <main className="pb-20">
       <DealDetails deal={deal} />
+
+      <Section
+        style={SECTION_STYLE.LIGHT}
+        width={SECTION_WIDTH.SM}
+        className="-translate-y-20 py-0"
```

Comment:
There might be a much cleaner way to the deal image to overlay on two different backgrounds. Not sure. I think about adding an after element to the image with z lower z-index to add the background that way instead of shifting everything up. I'm open if you have ideas!